MarsRover/Location.cs

```diff
@@ -0,0 +1,15 @@
+namespace MarsRover
+{
+    public class Location
+    {
+        public int X { get; set; }
+
+        public int Y { get; set; }

```

Comment:
May not want public setters. Could lead to some unexpected behaviours if you mutate `Location`s in the wrong way